en/docs/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway.md

```diff
@@ -20,4 +19,63 @@ To publish events to analytics cloud, you need to configure WSO2 API Manager to
     auth_token = “<use token that you generate>”
     ```
 
-3. Restart the API Manager and try to invoke APIs.
+2. Fill the auth token field with on-premise token that you obtained via Analytics Portal. If you didn't obtain the key
+ please follow 
+ [this]({{base_path}}/observe/api-manager-analytics/configure-analytics/register-for-analytics) 
+ documentation on how to obtain a token.
+
+3. Restart the API Manager, invoke APIs and log into Analytics Portal to view dashboards. Refer 
+[this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-overview) 
+documentation to
+ get more details about available dashboards and their usage.
+ 
+## Advanced Configurations
+ This section will explain the additional configurations you can make to fine tune the analytics data publishing
+ . Each of these configurations are set to a default value derived through testing. However, based on based other
+  factors there may be need to fine tune these parameters.
+  
+### Worker Thread Count
+This property denotes the number of threads that are publishing analytics data into analytics cloud. Default value is
+ one thread. One thread can serve up to 3200 request per second with unrestricted internet bandwidth. In case
+  single thread is not enough to meet the load handled by your gateway you will run into '**Event queue is full
+  . Starting to drop analytics events.**' error message in gateway logs. If you get this error during an API
+   invocation spike then please increase the 'Queue Size' as explained in next section. However, if you are getting
+    that error repeatedly then you should increase the worker thread count as shown below.
+   
+```toml
+[apim.analytics]
+enable = true
+config_endpoint = "https://analytics-event-auth.st.choreo.dev/auth/v1"
+auth_token = “<use token that you generate>”
+properties.'worker.thread.count' = 2
+```    
+
+### Queue Size
+This property denotes the number of analytics events Gateway keeps in-memory and used to handle request bursts
```

Comment:
Let's make Gateway title case everywhere for consistency.

Reply:
Fixed with 242c4b0